code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -236,13 +245,28 @@
 
 /datum/reagent/toxin/mindbreaker
 	name = "Mindbreaker Toxin"
-	description = "A powerful hallucinogen. Not a thing to be messed with. For some mental patients. it counteracts their symptoms and anchors them to reality."
+	description = "A powerful hallucinogen. Not a thing to be messed with. For some mental patients, it counteracts their symptoms and anchors them to reality."
 	color = "#B31008" // rgb: 139, 166, 233
 	toxpwr = 0
 	taste_description = "sourness"
+	metabolization_rate = REAGENTS_METABOLISM
+
+/datum/reagent/toxin/mindbreaker/liver_filter(mob/living/carbon/C, obj/item/organ/liver/L)
+	if(C.has_trauma_type(/datum/brain_trauma/mild/hallucinations))
+		. = FALSE
+	else
+		. = TRUE
 
 /datum/reagent/toxin/mindbreaker/on_mob_life(mob/living/carbon/M)
-	M.hallucination += 5
+	if(M.has_trauma_type(/datum/brain_trauma/mild/hallucinations))
+		// If the toxin is "anchoring" a patient, it lasts longer in their system
+		metabolization_rate = initial(metabolization_rate) * 0.2
```

Comment:
not terribly a fan of this since it ruins opportunities for reagents to manipulate other reagent's metab rates.